aspnetcore/fundamentals/http-logging/samples/6.x/Program.cs

```diff
@@ -1,26 +1,38 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
-
-namespace HttpLoggingSample
+using Microsoft.AspNetCore.HttpLogging;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+//builder.Services.AddRazorPages();
+#region Addservices
+builder.Services.AddHttpLogging(logging =>
+{
+    //Customize http logging here
+    logging.LoggingFields = HttpLoggingFields.All;
+    logging.RequestHeaders.Add("My-Request-Header");
+    logging.ResponseHeaders.Add("My-Response-Header");
+    logging.MediaTypeOptions.AddText("application/javascript");
+    logging.RequestBodyLogLimit = 4096;
+    logging.ResponseBodyLogLimit = 4096;
+
+});
+#endregion
+
+#region snippet
```

Comment:
Move `#region snippet` to the top, show entire file. Use highlighting.

Reply:
Is it still needed. since nested regions . It is a curious question.